packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -20,21 +19,33 @@ const STATUS = {
 
 type TDetailComponent = {
     document: {
-        onfido_name: string;
+        onfido_name?: string;
+        card: {
+            title: string;
+            description: string;
+            icon: string;
+        };
         details: {
+            fields: {
```

Comment:
Do we need this? :thinking: 
as far as I checked we don't use this

Reply:
@iman-fs, I believe this is fine, because [this](https://github.com/binary-com/deriv-app/pull/7115) PR was created due to the fact that people deleted code they thought wasn't being used. I believe the only issue should be getting the types right.

Reply:
@iman-fs this field is needed as it is pass into as a props from another component. It will give error if i remove this field and it might break the code also🙏